constrained-voucher.mkd

```diff
@@ -353,12 +353,18 @@ The Pledge discovers an IP address and port number that connects to the Registra
 
 No further discovery of hosts or port numbers is required, but a pledge that can do more than one kind of enrollment (future work offers protocols other than {{I-D.ietf-ace-coap-est}}), then a pledge may need to use CoAP Discovery to determine what other protocols are available.
 
-However, a Pledge that only supports the EST-coaps enrollment method SHOULD NOT use discovery for BRSKI resources, since it is more efficient to just try the supported enrollment method via the well-known BRSKI/EST-coaps resources, and it avoids the Pledge having to do complex CoRE Link Format parsing.
-
-In order to support this, the Registrar MUST support all of the EST resources at their default "well-known" locations (on the port number discovered) as well as any shorter form that the might also be supported.
-
-A Registrar SHOULD host any discoverable BRSKI resources on the same (UDP) server port that the Pledge's DTLS connection is using.
-This avoids the Pledge having to reconnect using DTLS, in order to access these resources.
+A Pledge that only supports the EST-coaps enrollment method SHOULD NOT use discovery for BRSKI resources.
+It is more efficient to just try the supported enrollment method via the well-known BRSKI/EST-coaps resources.
+This also avoids the Pledge doing any CoRE Link Format parsing, which is specified in {{I-D.ietf-ace-coap-est, Section 4.1}}.
+
+In order to support this, the Registrar MUST support all of the EST resources at their default ".well-known" locations (on the port number discovered) 
+as well as any server-specific shorter form that might also be supported.
+
+However, when discovery is being done by the Pledge, it is possible for the Registrar to return references to resources which are on different port numbers.
+The Registrar MUST NOT do this because a Pledge that is connected via a Join Proxy can only access a single UDP port.
```

Comment:
```suggestion
The Registrar SHOULD NOT use different ports numbers by default, because a Pledge that is connected via a Join Proxy can only access a single UDP port.
A Registrar connecting using a technology that does not support Join Proxies MAY be configured to use multiple port numbers.
```